Cargo.toml

```diff
@@ -5,4 +5,6 @@ version = "0.1.0"
 authors = ["Corey Richardson <corey@resin.io>"]
 
 [dependencies]
-serde_json = "1.0"
\ No newline at end of file
+serde = "^1.0"
+serde_json = "^1.0"
+nom = "^3.0"
```

Comment:
No need for the caret, since it is the default one: `nom = "^3.0"` is equivalent to `nom = "3.0"`.